docs/deployment/containers/docker.txt

```diff
@@ -219,6 +219,8 @@ Read about Docker Compose specifics `here <https://docs.docker.com/compose/>`_.
         - /tmp/crate/01:/data
       command: >
         crate -Cnetwork.host=_site_
+        -Cdiscovery.seed_hosts=crate02,crate03
+        -Ccluster.initial_master_nodes=crate02,crate03
```

Comment:
also the local node should be included in `initial_master_nodes` to also be part of the initial master voting.

Reply:
I suppose for all nodes?

Reply:
yep